src/ucp/proto/proto_common.c

```diff
@@ -22,6 +22,14 @@ ucp_proto_common_get_rsc_index(const ucp_proto_init_params_t *params,
     return params->ep_config_key->lanes[lane].rsc_index;
 }
 
+size_t
+ucp_proto_common_get_seg_size(const ucp_proto_common_init_params_t *params,
+                              ucp_lane_index_t lane)
```

Comment:
1. did it pass clang-format?
2. can it be static

Reply:
yes (to both questions)